app/grandchallenge/algorithms/tasks.py

```diff
@@ -91,33 +104,25 @@ def run_algorithm_job_for_inputs(*, job_pk, upload_pks, user_upload_pks):
                 for civ_pk, upload_pk in user_upload_pks.items()
             ]
         )
-        start_jobs = chord(file_tasks, start_jobs).on_error(
+        start_job = chord(file_tasks, start_job).on_error(
             on_job_creation_error.s(job_pk=job_pk)
         )
-    on_commit(start_jobs.apply_async)
+    on_commit(start_job.apply_async)
 
 
 @shared_task(bind=True)
 def on_job_creation_error(self, task_id, *args, **kwargs):
     job_pk = kwargs.pop("job_pk")
     job = Job.objects.get(pk=job_pk)
 
-    # Send an email to the algorithm editors and creator on job failure
+    # Send email to creator on job failure
     linked_task = send_failed_job_notification.signature(
         kwargs={"job_pk": job.pk}, immutable=True
     )
 
     error_message = ""
-    missing_inputs = list(civ for civ in job.inputs.all() if not civ.has_value)
-    if missing_inputs:
-        error_message += (
-            f"Job can't be started, input is missing for "
-            f"{oxford_comma([c.interface.title for c in missing_inputs])}"
-        )
-
     res = self.AsyncResult(task_id).result
-
-    if isinstance(res, ImageImportError):
```

Comment:
Did this ever work? `res` is a instance of a `ChordError`. I couldn't find a better way than string matching so far...

Reply:
It must have done at some point, but obviously wasn't covered by a test. Celery has this annoying feature of auto-upgrading chains of groups to chords, which changes their return types. Must have happened when changing the implementation of the image import tasks.